packages/record/src/recorder.spec.ts

```diff
@@ -119,6 +122,371 @@ test("Recorder handleDataAvailable", () => {
   expect(Data.LookitS3.prototype.onDataAvailable).toHaveBeenCalledTimes(1);
 });
 
+test("Recorder insert webcam display without height/width", () => {
+  // Add webcam container to document body.
+  const webcam_container_id = "webcam-container";
+  document.body.innerHTML = `<div id="${webcam_container_id}"></div>`;
+  const webcam_div = document.getElementById(
+    webcam_container_id,
+  ) as HTMLDivElement;
+
+  const jsPsych = initJsPsych();
+  const rec = new Recorder(jsPsych, "prefix");
+
+  const media = {
+    stream: { getTracks: jest.fn().mockReturnValue([{ stop: jest.fn() }]) },
+  };
+  jsPsych.pluginAPI.getCameraRecorder = jest.fn().mockReturnValue(media);
+
+  // Should add the video element with webcam stream to the webcam container.
+  rec.insertWebcamFeed(webcam_div);
+
+  // Use the HTML template and settings to figure out what HTML should have been added.
+  const height: CSSWidthHeight = "auto";
+  const width: CSSWidthHeight = "100%";
+  const webcam_element_id: string = "lookit-jspsych-webcam";
+  const params = { height, width, webcam_element_id };
+  let rendered_webcam_html = Mustache.render(webcamFeed, params);
+
+  // Remove new lines, indents (tabs or spaces), and empty HTML property values.
+  rendered_webcam_html = rendered_webcam_html.replace(
+    /(\r\n|\n|\r|\t| {4})/gm,
+    "",
+  );
+  let displayed_html = document.body.innerHTML;
+  displayed_html = displayed_html.replace(/(\r\n|\n|\r|\t| {4})/gm, "");
+  displayed_html = displayed_html.replace(/(="")/gm, "");
```

Comment:
Do you only have to remove white space in tests?

Reply:
Yes this is only relevant in tests. The problem here was me trying to figure out how to compare the expected HTML that's generated from Mustache with the actual HTML that I get from `document.body.innerHTML`.
Here's what I get from `Mustache.render`:
```html
<video 
          autoplay 
          playsinline 
          muted 
          id="lookit-jspsych-webcam" 
          width="100%" 
          height="auto" 
          style="display:inline-block;">
</video>
```
And here's what I get from `document.body.innerHTML`:
```html
<div id="webcam-container"><video autoplay="" playsinline="" muted="" id="lookit-jspsych-webcam" width="100%" height="auto" style="display:inline-block;"></video></div>
```
So the important thing is that the content from `Mustache.render` appears in the document HTML, but the comparison between these is tricky because of the spaces/new lines, and thea ddition of `=""` for properties with no values.
I'm happy to change this if there's a better to test the same thing!